README.md

```diff
@@ -15,7 +16,186 @@ deprecation policy, and this may be subject to change without notice.
 The API this module exports is experimental and is not covered by any
 deprecation policy, and may be changed without notice.
 
+
+## Prerequisites
+1. Your application will need to be using Node.js version 6.12.3 or greater, or Node.js 8.9.4 or greater.
```

Comment:
The text in *.md files should be formatted to fit 80 columns where possible.

Reply:
Done.